caddyhttp/proxy/proxy.go

```diff
@@ -273,12 +273,18 @@ func mutateHeadersByRules(headers, rules http.Header, repl httpserver.Replacer)
 	for ruleField, ruleValues := range rules {
 		if strings.HasPrefix(ruleField, "+") {
 			for _, ruleValue := range ruleValues {
-				headers.Add(strings.TrimPrefix(ruleField, "+"), repl.Replace(ruleValue))
+				var replacement = repl.Replace(ruleValue)
```

Comment:
Stylistic nit-pick: change to `replacement := ...` rather than using the `var` longform. (Same below)